File: pinned/lib/models/test_score_model.dart
import 'package:flutter/material.dart';

class TestScoreModel {
  final int id;
  final String email;
  final int score;
  final DateTime createdAt;

  TestScoreModel(
      {required this.id,
      required this.email,
      required this.score,
      required this.createdAt});
}

File: pinned/lib/models/test_score_model.js
const mongoose = require('mongoose');

const UserSchema = new mongoose.Schema({
  id : {type: Number , required: true, unique: true},
  email: { type: String, required: true, unique: true },
  name: { type: Number, required: true },
  createdAt: { type: Date, required: true },
});

module.exports = mongoose.model('TestDB', UserSchema);